internal/usage/usage.go

```diff
@@ -20,6 +20,9 @@ const (
 	Profile                         = "Profile to use from your configuration file."
 	Members                         = "Number of members in the replica set."
 	Shards                          = "Number of shards in the cluster."
+	ProcessName                     = "The unique identifier for the host of a MongoDB process in the following format: {hostname}:{port}."
```

Comment:
point on that we may need to split those commands on one for atlas one for om/cm so atlas can have a validation on format

Reply:
Can I just run the validation if we have `config.Service() == config.CloudService`?

Reply:
fine by me but consider the change of the flag name as well, `processName` for atlas vs `hostId` for OM/CM and flagging the required flag accordingly